cluster/providers.tf

```diff
@@ -3,8 +3,10 @@
 #
 
 provider "aws" {
-  region = "us-west-2"
-  version = "~> 1.24"
+  region                  = "${var.region}"
+  shared_credentials_file = "~/.aws/credentials"
```

Comment:
I would omit this key so that it will still default to the full credential chain.